src/rtc_peer.ts

```diff
@@ -243,23 +249,7 @@ export class RTCPeer extends EventEmitter {
     }
 
     private onTrack(ev: RTCTrackEvent) {
-        if (this.pc && ev.track.kind === 'video') {
-            // We force the transceiver direction of the incoming screen track
-            // to be 'sendrecv' so Firefox stops complaining.
-            // In practice the transceiver is only ever going to be used to
-            // receive.
-            for (const t of this.pc.getTransceivers()) {
-                if (t.receiver && t.receiver.track === ev.track) {
-                    if (t.direction !== 'sendrecv') {
-                        this.logger.logDebug('RTCPeer.onTrack: setting transceiver direction for track');
-                        t.direction = 'sendrecv';
-                    }
-                    break;
-                }
```

Comment:
From what I could tell, this workaround isn't needed anymore as we default to `sendrecv`. See https://github.com/mattermost/calls-common/pull/47/files#r2012726703